src/DssSpell.t.sol

```diff
@@ -528,81 +508,86 @@ contract DssSpellTest is DssSpellTestBase {
+    function testVestMKR() public {
         VestAbstract vest = VestAbstract(addr.addr("MCD_VEST_MKR_TREASURY"));
-        assertEq(vest.ids(), 23);
+        assertEq(vest.ids(), 24);
 
         uint256 prevAllowance = gov.allowance(pauseProxy, addr.addr("MCD_VEST_MKR_TREASURY"));
 
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        assertEq(gov.allowance(pauseProxy, addr.addr("MCD_VEST_MKR_TREASURY")), prevAllowance + 803 * WAD);
+        assertEq(gov.allowance(pauseProxy, addr.addr("MCD_VEST_MKR_TREASURY")), prevAllowance + 787.70 ether);
 
         assertEq(vest.cap(), 1_100 * WAD / 365 days);
-        assertEq(vest.ids(), 24);
+        assertEq(vest.ids(), 28);
 
-        // Friday,   1 July 2022 00:00:00 UTC
-        uint256 JUL_01_2022 = 1656633600;
-        // Saturday, 1 July 2023 00:00:00 UTC
-        uint256 JUL_01_2023 = 1688169600;
+        uint256 AUG_01_2022 = 1659326400;
+        uint256 AUG_01_2023 = 1690862400;
+        uint256 SEP_28_2022 = 1664337600;
+        uint256 SEP_28_2024 = 1727496000;
 
-        address GRO_WALLET = wallets.addr("GRO_WALLET");
+        address GOV_WALLET1 = 0xbfDD0E744723192f7880493b66501253C34e1241;
+        address GOV_WALLET2 = 0xbb147E995c9f159b93Da683dCa7893d6157199B9;
+        address GOV_WALLET3 = 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73;
+        address SNE_WALLET = wallets.addr("SNE_WALLET");
 
         // -----
-        assertEq(vest.usr(24), GRO_WALLET);
-        assertEq(vest.bgn(24), JUL_01_2022);
-        assertEq(vest.clf(24), JUL_01_2022);
-        assertEq(vest.fin(24), JUL_01_2022 + 365 days);
-        assertEq(vest.fin(24), JUL_01_2023);
-        assertEq(vest.mgr(24), address(0));
-        assertEq(vest.res(24), 1);
-        assertEq(vest.tot(24), 803 * WAD);
-        assertEq(vest.rxd(24), 0);
-
-        uint256 prevBalance = gov.balanceOf(GRO_WALLET);
-
-        // Give admin powers to test contract address and make the vesting unrestricted for testing
-        giveAuth(address(vest), address(this));
-        vest.unrestrict(24);
-
-        // 20220907 exec: Warp 1/2 year since cliff, ensure vest was accumulated
-        hevm.warp(JUL_01_2022 + 365 days / 2);
-        vest.vest(24);
-        assertEq(gov.balanceOf(GRO_WALLET), prevBalance + (803 * WAD / 2));
-
-        // 20220907 exec: Warp to end and vest remaining
-        hevm.warp(JUL_01_2022 + 365 days);
-        vest.vest(24);
-        assertEq(gov.balanceOf(GRO_WALLET), prevBalance + 803 * WAD);
-
-        // 20220907 exec: Warp even further and make sure nothing else was accumulated
-        hevm.warp(JUL_01_2022 + 365 days + 10 days);
-        vest.vest(24);
-        assertEq(gov.balanceOf(GRO_WALLET), prevBalance + 803 * WAD);
+        assertEq(vest.usr(25), GOV_WALLET1);
+        assertEq(vest.bgn(25), AUG_01_2022);
+        assertEq(vest.clf(25), AUG_01_2023);
+        assertEq(vest.fin(25), AUG_01_2023);
```

Comment:
`fin` could be checked against `bgn` + x days to ensure duration correctness, following previous patterns